files/es/learn/common_questions/thinking_before_coding/index.html

```diff
@@ -58,12 +58,12 @@ <h3 id="¿Qué_es_exactamente_lo_que_quiero_lograr">¿Qué es exactamente lo que
 
 <p>Esta es la más importante pregunta a responder, ya que impulsa todo lo demás. Enumera todas la metas que deseas alcanzar. Puede ser cualquier cosa: vender bienes para ganar dinero, expresar opiniones políticas, conocer nuevos amigos, actuar con músicos, coleccionar imágenes de gatos o lo que quieras.</p>
 
-<p>Supongamos que eres músico. Y puedes desear que:</p>
+<p>Supongamos que eres músico. Podrías desear:</p>
 
 <ul>
- <li>Permitir que las gente escuche tu música.</li>
- <li>Venta de golosinas.</li>
- <li>Conocer otros músicos.</li>
+ <li>Posibilitar que la gente escuche tu música.</li>
+ <li>Vender golosinas.</li>
```

Comment:
Tú no las vendes, hay venta de

```suggestion
 <li>Venta de golosinas.</li>
```

Reply:
@Graywolf9, acá tengo un problema: el "título" de la lista en cuestión dice: "Supongamos que eres músico. Y puedes desear:"; "Venta de" no es algo que "puedes desear"; debería ir en infinitivo, y por eso creo que lo correcto sería "vender". Pero bueno, como vos digas, espero tu respuesta, y si considerás que es "Venta de", hago el Commit.